src/modules/render.js

```diff
@@ -0,0 +1,26 @@
+import apiURL from './api.js';
+
+const scoreList = document.getElementById('scoreList');
+// const score = document.getElementById('score');
+
+export default async function renderHTML() {
```

Comment:
- [ ]  You should use the arrow functions instead of the function keyword. Using the arrow function will be the right way of applying ES6 javascript.

Reply:
@ishimwezachee Thank you for the review, I have applied the new changes to the code, and I hope it works this time.